cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -168,6 +169,29 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
   }
 
+  @Test def create_lookup_index() {
+    // remove autogenerated lookup indexes
+    val res = execute("SHOW LOOKUP INDEXES YIELD name")
+    res.columnAs[String]("name").foreach(n => execute(s"DROP INDEX $n"))
+
+    testQuery(
+      title = "Create a lookup index for nodes",
+      text = "A named lookup index for all nodes that have any label can be created with `CREATE LOOKUP INDEX index_name FOR (n) ON EACH labels(n)`. " +
```

Comment:
```suggestion
      text = "A named lookup index for all nodes with a label can be created with `CREATE LOOKUP INDEX index_name FOR (n) ON EACH labels(n)`. " +
```

Reply:
I think I prefer `any label` over `a label`, since it can have more than one as well
```suggestion
      text = "A named lookup index for all nodes with any label can be created with `CREATE LOOKUP INDEX index_name FOR (n) ON EACH labels(n)`. " +
```

Reply:
Then I would suggest the following, as _any_ doesn't really indicate possible plural.
```suggestion
      text = "A named lookup index for all nodes with one or more labels can be created with `CREATE LOOKUP INDEX index_name FOR (n) ON EACH labels(n)`. " +
```